share/completions/rcctl.fish

```diff
@@ -0,0 +1,4 @@
+
+complete -c rcctl -xa 'check ls reload restart stop start disable enable' -n 'not __fish_seen_subcommand_from list check ls reload restart stop start enable disable'
+complete -c rcctl -n '__fish_seen_subcommand_from check reload restart stop start enable disable' -xa '( ls /etc/rc.d )'
```

Comment:
Instead of `ls`, use `printf`. I.e. `(set -l files /etc/rc.d/*; printf "%s\n" $files)`.